okta/komand_okta/actions/send_push/action.py

```diff
@@ -0,0 +1,40 @@
+import komand
+from .schema import SendPushInput, SendPushOutput, Input, Output, Component
+# Custom imports below
+import requests
+
+
+
+class SendPush(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name='send_push',
+            description=Component.DESCRIPTION,
+            input=SendPushInput(),
+            output=SendPushOutput())
+
+    def run(self, params={}):
+        self.logger.info("Sending a push verify to %s", params)
+        user_id = params.get(Input.USER_ID)
+        factor_id = params.get(Input.FACTOR_ID)
+
+        okta_url = self.connection.okta_url
+        url = requests.compat.urljoin(okta_url, f'/api/v1/users/{user_id}/factors/{factor_id}/verify')
+        response = self.connection.session.post(url)
+        if response.status_code == 401:
+            self.logger.error('Okta: Invalid token or domain')
+
+        data = response.json()
```

Comment:
Same as try/except comment in other action file

Reply:
👍